modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v7_0_0/CommerceAddressUpgradeProcess.java

```diff
@@ -155,8 +154,9 @@ private void _setPhoneNumber(Address address, String phoneNumber) {
 		ListType listType = _listTypeLocalService.getListType(
 			"phone-number", ListTypeConstants.ADDRESS_PHONE);
 
-		ServiceContext serviceContext =
-			ServiceContextThreadLocal.getServiceContext();
+		ServiceContext serviceContext = new ServiceContext();
+
+		serviceContext.setUserId(address.getUserId());
```

Comment:
It seems that the only thing the `serviceContext` is needed for inside the service method is getting the `uuid`. We can probably just pass in the `userId` to the service method directly off of the `address` instance, and then use `PortalUUIDUtil.generate()` to set the `uuid` onto the `serviceContext` instance.